vms/platformvm/api/static_service.go

```diff
@@ -98,6 +98,9 @@ type Staker struct {
 	StakeAmount *json.Uint64 `json:"stakeAmount,omitempty"`
 }
 
+// GenesisValidator should to be used for genesis validators only.
+type GenesisValidator Staker
```

Comment:
What do we need this alias for? I feel like we should just use `Staker` if we're not going to add any different functionality

Reply:
Pending introduction of ACP20, we will change serialization on NodeID in Staker, but we will never change serialization of GenesisValidator (since then genesisID will change).
This PR prepare that change and clearly separate the types.
I reckon it does not really bring much value so far, but it should in the near future.

Reply:
The type is changed in a followup